resources/user-agents/crawlers/copyright-plagiarism/copyright-plagiarism.json

```diff
@@ -73,6 +73,13 @@
           "properties": {
             "Browser": "TutorGig"
           }
+        },
+        {
+          "match": "TurnitinBot*",
```

Comment:
There is already a file for this bot (crawlers/turnitinbot.json). Please move this pattern to that file and move the file to this folder.